cmd/lint.go

```diff
@@ -69,10 +71,44 @@ func validateSourceCommandAction(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return fmt.Errorf("locating package root failed: %w", err)
 	}
+
+	err = validateSpecVersion(packageRootPath)
+	if err != nil {
+		return err
+	}
+
```

Comment:
Maybe, this can be moved after `validator.ValidateFromPath()` to show the linting errors that could be in the package?